### PR TITLE
feat: add LEAR and tutorial

### PR DESCRIPTION
**Pull request by wltjr1007**
[Colab notebook](https://colab.research.google.com/drive/1m16h7rprsCdXavNbooXFhK7IPh-OwcF2#scrollTo=ccL8Vtnwrv9u) for your convinience.
(Only difference between the markdown tutorial and this jupyter notebook is the PnPXAI's github repository)

```bash
# Markdown
!git clone --quiet https://github.com/OpenXAIProject/pnpxai
```
to
```bash
# Jupyter
!git clone --quiet -b feature/lear https://github.com/wltjr1007/pnpxai
```

### CODE REVIEW

File: pnpxai/explainers/lear.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    """
+    LEAR Explainer class for generating counterfactual explanations.
+
+    Args:
+        model_name (str): The name of the pre-trained model to load.
+        device (str, optional): Device to use for computation ('cuda' or 'cpu').
+        model (torch.nn.Module, optional): This is not used for LEAR. It is included for compatibility with the base class.
+    """
+
+    def __init__(self, model_name="wltjr1007/LEAR", device=None, model=None):
```
Comment (enver1323) [inline pnpxai/explainers/lear.py:16]: Everything looks great! Could we only, please, rearrange the parameter order for better alignment between all other explainers like this?
```suggestion
    def __init__(self, model=None, model_name: str="wltjr1007/LEAR", device: torch.device=None):
```

**Review by enver1323 — APPROVED** [review]
Everything looks great. Just had one suggestion for better consistency.

File: pnpxai/explainers/lear.py (hunk 1 above)
Comment (wltjr1007) [reply pnpxai/explainers/lear.py]: Thanks. Just comitted your suggestion :)

Comment (enver1323) [reply pnpxai/explainers/lear.py]: Thank you!

**Pull request merged**